guacamole/src/main/webapp/index.html

```diff
@@ -19,7 +19,8 @@
 -->
 <html ng-app="index" ng-controller="indexController">
     <head>
-        <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+        <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
+        <meta http-equiv="x-ua-compatible" content="IE=edge"/>
```

Comment:
Is it "x-ua-compatible" or "X-UA-Compatible"?

Reply:
According to the MSDN page, all lower-case:

https://msdn.microsoft.com/en-us/library/jj676915(v=vs.85).aspx

Reply:
Ah, OK.